ModelBundle/Tests/Functional/Repository/ContentRepositoryTest.php

```diff
@@ -256,6 +256,9 @@ public function provideFindOneByContentIdAndLanguageAndVersion()
      */
     public function testFindByContentTypeAndSiteIdInLastVersionForPaginate($contentType, $descriptionEntity, $search, $siteId, $skip, $limit, $count)
     {
+        if (isset($search['columns']['status_label'])) {
+            $this->markTestSkipped();
```

Comment:
Perfect if you can add a comment to get that test back when the related problem is solved